src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -23,13 +29,24 @@
 	@Option( names = "--list", description = "List all ignored elements." )
 	private boolean list;
 
-	@Parameters( arity = "1", description = "Exactly one test report." )
+	@Parameters( arity = "0..1", description = "Exactly one test report." )
 	private File testReport;
 
 	@Override
 	public void run() {
-		// TODO Implement.
-		logger.info( "Not yet implemented!" );
+		if ( list ) {
```

Comment:
Would try to avoid that deep nesting:

```java
if ( !list ) {
    return;
}

final Path ignoreFile = projectConfig.resolve( IGNORE_FILE );
if ( !ignoreFile.toFile().exists() ) {
    logger.error( "Ignore file '{}' does not exist.", ignoreFile.getPath() );
}

// ...
```

Reply:
Either way, this needs to be modified for the next ignore implementation.